.travis.yml

```diff
@@ -10,6 +10,7 @@ branches:
  only:
  - master
  - devel
+ - gapfill_bugfix
```

Comment:
Please remove this line. Your pull request will be tested on Travis anyway because it is a PR for the devel branch.